android/engine/src/main/java/org/smartregister/fhircore/engine/ui/register/ComposeRegisterFragment.kt

```diff
@@ -48,21 +48,23 @@ abstract class ComposeRegisterFragment<I : Any, O : Any> : BaseRegisterFragment<
           val pagingItems = registerData.value.collectAsLazyPagingItems()
           val showResultsCount by registerDataViewModel.showResultsCount.observeAsState(false)
           val showLoader by registerDataViewModel.showLoader.observeAsState(false)
-
-          if (showLoader) LoaderDialog(modifier = Modifier)
+          val _modifier = Modifier
```

Comment:
Rename this variable to **modifier** to follow kotlin conventions

Reply:
modifier is required as its passing to internal components also i.e body() here, I have renamed _modifier -> modifier